packages/web-components/@sfx/sayt/stories/index.ts

```diff
@@ -1,25 +1,48 @@
 import { storiesOf } from '@storybook/html';
 import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+
 import '../src/index.ts';
+import { getDisplayCode } from '../../../../../.storybook/common';
 
-const autocompleteDataReceivedEvent = new CustomEvent('autocomplete_received_results',
+// @TODO allow for sending event with searchbar ID. This should allow for one
+// story's events to not affect another story.
+const autocompleteDataReceivedEvent = new CustomEvent('sfx::autocomplete_received_results',
   { detail: [
       { "title": "Brands", "items": [{ "label": "Cats" }, { "label": "Dogs" }] },
       { "title": "default", "items": [{ "label": "Cars" }, { "label": "Bikes" }] }
     ],
     bubbles: true }
 );
 
+function getSayt(searchbar='', showSayt=true): string {
```

Comment:
should there be spacing after the = here?

Reply:
In my opinion: no, but that doesn't mean it's cool to do in JavaScript. Will let someone else add their input.

Reply:
I expect that our linter settings will require padding.

Reply:
I've only worked on one project to reference, but we **did** include spacing.
I also checked MDN, and the example they provided includes spacing.